src/test/mochitest/browser_dbg-pause-ux.js

```diff
@@ -0,0 +1,44 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+function getScrollTop(dbg) {
+  return getCM(dbg).doc.scrollTop;
+}
+
+async function waitForMatch(dbg, { matchIndex, count }) {
+  await waitForState(dbg, state => {
+    const result = dbg.selectors.getFileSearchResults(state);
+    return result.matchIndex == matchIndex
+      && result.count == count;
+  })
+}
+
+add_task(async function() {
+  const dbg = await initDebugger("doc-scripts.html");
+
+  // Make sure that we can set a breakpoint on a line out of the
+  // viewport, and that pausing there scrolls the editor to it.
+  let longSrc = findSource(dbg, "long.js");
+  await addBreakpoint(dbg, longSrc, 66);
+  invokeInTab("testModel");
+  await waitForPaused(dbg);
+
+  const pauseScrollTop = getScrollTop(dbg);
+
+  log("1. adding a breakpoint should not scroll the editor");
+  getCM(dbg).scrollTo(0, 0);
+  await addBreakpoint(dbg, longSrc, 11);
+  is(getScrollTop(dbg), 0, "scroll position");
+
+  log("2. searching should jump to the match");
+  pressKey(dbg, "fileSearch");
+  type(dbg, "check");
+  await waitForMatch(dbg, { matchIndex: 0, count: 2 });
+  const matchScrollTop = getScrollTop(dbg);
+  debugger;
+  // ok(isVisibleInEditor(dbg, findElement(dbg, "gutter", 27)), "search match is visible");
+  ok(pauseScrollTop != matchScrollTop, "did not jump to debug line");
+
+  // await waitForever(dbg);
+  // debugger
```

Comment:
not done yet, but i like the idea of adding a test that performs some basic tasks like searching, adding breakpoints when we're paused... and makes sure nothing stupid happens